app/views/onboarding/Onboarding1.js

```diff
@@ -58,35 +58,38 @@ class Onboarding extends Component {
                 zIndex: 10,
               }}>
               <NativePicker
+                languagePicker // pass picker type to handle RTL checking because we can reuse this picker for any type not language picker only
```

Comment:
Ideally, language specific stuff should be in the onValueChanged handlers.

Check #557 with upcoming changes. If you keep your language stuff in onValueChanged then both changes can coexist.

Reply:
Translation logic removed from the Picker